mesonbuild/build.py

```diff
@@ -2152,8 +2152,8 @@ def __init__(self, name, subdir, subproject, for_machine: MachineChoice, sources
         super().__init__(name, subdir, subproject, for_machine, sources, objects, environment, kwargs)
         self.typename = 'shared module'
 
-    def get_default_install_dir(self, environment):
-        return environment.get_shared_module_dir()
+    def get_default_install_dir(self, environment) -> T.Tuple[str, str]:
+        return environment.get_shared_module_dir(), 'moduledir_shared'
```

Comment:
Did you forget the `{}` here, or is this on purpose?

Reply:
Forgot.